TFT/src/User/API/interfaceCmd.c

```diff
@@ -310,7 +310,21 @@ void sendQueueCmd(void)
             }
             break;
           }
-
+          case 710: //M710
+          {
+            if(cmd_seen('S'))
+            {
+              fanSetSpeed(1, cmd_value()); 
+            }
+            else
+            {
+              char buf[12];
+              sprintf(buf, "S%d\n", fanGetSpeed(1));
+              strcat(infoCmd.queue[infoCmd.index_r].gcode,(const char*)buf);
+              fanSetSendWaiting(1, false);
+            }
+            break;
+          }

```

Comment:
does M710 exist in marlin firmware?

Reply:
Doesn't look like it:

https://marlinfw.org/meta/gcode/

Reply:
That's true: M710 is not in official marlin repo (yet). Sorry, accidentally commit that change. It is now removed from master. Thanks for the note.